reference_guide/intellij_artifacts.md

```diff
@@ -0,0 +1,107 @@
+---
+title: IntelliJ Platform Artifacts Repository
+---
+
+JetBrains maintains a public repository that hosts artifacts related to the IntelliJ Platform, such as binaries and source code. This 
+repository makes artifacts more accessible for plugin authors.
+
+The IntelliJ Platform Artifacts Repository contains:
+* [Releases repository](https://www.jetbrains.com/intellij-repository/releases/) for release and EAP versions.
+* [Snapshots repository](https://www.jetbrains.com/intellij-repository/snapshots/) for nightly successful 
+  builds of each branch.
+  
+More artifacts will be hosted in the future.
+
+[Dependencies](https://jetbrains.bintray.com/intellij-third-party-dependencies) of individual modules from the IntelliJ
+Platform are hosted at the bintray repository. These artifacts should not be used directly, but a link to this 
+repository should be added to pom.xml/build.gradle files if individual modules from the IntelliJ Platform Artifacts Repository are used.    
+
+### How to Use the IntelliJ Platform Artifacts Repository
+There are two steps to using the repository:
+1. Add the corresponding repository URL to a Maven or Gradle script.
+2. Specify the Maven coordinates for the artifact. 
```

Comment:
Could we specify here and everywhere else that the main method of consuming those repositories is https://github.com/JetBrains/gradle-intellij-plugin and not directly? cc @chashnikov

Reply:
@shalupov  - Thanks for your review.
Yes, I can clarify. I'll add a lead-in sentence to the effect that the two steps describe using the repositories by adding information to a project's Gradle build script, which is supported by gradle-intellij-plugin.

Reply:
It loos like this wasn't fully resolved. Currently we have
> Artifacts in the repositories are utilized by adding information to a project’s build.gradle file. See the gradle-intellij-plugin for more information about Gradle support.

This may be misleading. You don't need to use gradle-intellij-plugin to access artifacts for individual modules, it won't help you with that. Actually we have two kinds of artifacts in there repositories. The first one is full zip distributions of IntelliJ IDEA (and theirs sources). These artifacts aren't supposed to be accessed directly, they are used inside gradle-intellij-plugin (see [my comment](https://github.com/JetBrains/intellij-sdk-docs/pull/109#discussion_r214303721) in this pull request). The second one is individual modules from IntelliJ Platform. They may be used directly and they aren't quite related to gradle-intellij-plugin.

Maybe we should simply remove description of the first kind of artifacts from the public page? (The first two items in 'Specify the Artifact' section, and 'Gradle Example for IntelliJ IDEA Community Edition Distribution' section.) And replace them by a small note like "These repositories also provides cross-platform full distributions of IntelliJ IDEA which are used by gradle-intellij-plugin and aren't supposed to be accessed directly"? This will greatly simplify the document.

@JohnHake @zolotov @shalupov what do you think?

Reply:
+1 for removing the first kind of artifacts from the public page and for adding `These repositories also provides cross-platform full distributions of IntelliJ IDEA which are used by gradle-intellij-plugin`.

Not sure about `aren't supposed to be accessed directly`, as they still can be downloaded directly if needed modules are not published separately yet.

Reply:
@JohnHake so what do you think about this?